### PR TITLE
Refs #10970 Fixing specs

### PR DESCRIPTION
**Pull request by cpeters**

### CODE REVIEW

File: test/support/vcr.rb [hunk 1]
```diff
@@ -2,6 +2,8 @@
 require "vcr"
 require "active_support/concern"
 
+WebMock.disable_net_connect!(:allow_localhost => true)
```
Comment (komidore64) [inline test/support/vcr.rb]: this isn't part of the specs. do we want to include this? because it will affect all tests.

Comment (cpeters) [inline test/support/vcr.rb]: Yes, it's required to get all the specs to run and pass.

Comment (komidore64) [inline test/support/vcr.rb]: oh ok

**Comment by komidore64** [comment]
ack

**Comment by cpeters** [comment]
@ehelms @jlsherrill Can you guys review this? It seems like the default behavior of webmock has changed and by default no longer allows any connections.

**Comment by jlsherrill** [comment]
ACK, those connections are to elasticsearch and its not worth doing much about now (as they don't cause a problem).  @cpeters is opening an issue to remove it once ES is gone

**Comment by komidore64** [comment]
awesome thanks @jlsherrill!

**Pull request merged**